Tutorials/PartIII_solution/iLeakyReLU/iLeakyReLUParser.py

```
@@ -0,0 +1,43 @@
+
+# (in Generic/Parsers.py the following imports already exist:
+#  import math; import numpy as np; import onnx_graphsurgeon as gs;
+#  from Deeploy.DeeployTypes import NodeParser, NetworkContext)
+
+
+class iLeakyReLUParser(NodeParser):
+
+    def __init__(self):
+        super().__init__()
+
+    def parseNode(self, node: gs.Node) -> bool:
```

Ruff (0.16.0)

[error] Undefined name `gs`

(F821)